main.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  main.swift
+//  Text Based Adventure
+//
+//  Created by Tanya Burke on 10/3/19.
+//  Copyright © 2019 Tanya Burke. All rights reserved.
+
+//
+
+import Foundation
+
+var isPlaying = false
+var currentLevel = 1
+var finalLevel = 10
+repeat {
+
+       if currentLevel == finalLevel {
+         print("I hope you enjoyed your adventure")
+         isPlaying = false // stops the game
+    } else if currentLevel == 1{
+        print("Your adventure begins!")
+        currentLevel += 1
+        print("Pick a number from 1 to 100")
+
+        let userEnteredNumAsString = readLine()
+        let userEnteredNumAsInt = Int(userEnteredNumAsString!) ?? 3
+        if userEnteredNumAsInt % 2 == 0{
+          print ("I love even numbers, do you?")
+           print("Now let's figure out what path you will choose, It's Friday night what would you like to do...Choose one \"Movie\" or \"Date\" or \"Study\"")
+            let movieOrDateOrStudy = readLine()!.lowercased()
+            print("You decide to have companion join you, who will it be a \"hot girl\" or a \"hot guy\"")
+                    let yourCompanion = readLine()!.lowercased
+                    print(yourCompanion() == "hot girl" ? "she is excited to join you" : "he is happy to hang out with you")
+    
+                    if movieOrDateOrStudy == "Movie".lowercased(){
+                        print("What type of movie would you and \(yourCompanion) like to see? \"Action\" or \"Horror\" or \" Fantasy\"")
+                        let movieType = readLine()
+                        switch movieType{
+                        case "Action":
```

Comment:
Please keep in mind that this won't go into effect unless the user types it exactly as is, a better use would be to lowercase the case manually and to use the .lowercased function.